prod.Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM node:18-alpine as builder
```

Comment:
Remove `builder` build stage, instead:
1. Start from oven/bun:alpine as before
2. Copy first the manifest and lockfile
3. Bun install like below
4. Copy all assets
5. :heavy_check_mark: